halotools/mock_observables/two_point_clustering/angular_tpcf.py

```diff
@@ -205,13 +206,14 @@ def area_spherical_cap(chord):
             D2R = N2*(da*rho2)
 
             # calculate the random-random pairs.
+            # cbx: This should be NR(NR-1) if we are doing an autocorr
```

Comment:
The change/conversation https://github.com/manodeep/Corrfunc/pull/201 explains it in full I think

Reply:
Hmm, are other autocorrelation function results impacted by this besides angular_tpcf?

Reply:
I went ahead and raised a separate issue #974 since this was the only lingering issue and this is a solid PR that's otherwise good to go.